content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,9 @@ weight: 10
 ## Security
 
 - **Added** the [`v1beta1` authorization policy model](/docs/concepts/security#authorization) for enforcing access control. This will eventually replace the [`v1alpha1` RBAC policy](/docs/reference/config/authorization/istio.rbac.v1alpha1/).
+- **Added** experimental support for [automatic mutual TLS](docs/tasks/security/authentication/auto-mtls/).
+- **Added** experimental [DNS Certificate Management](/blog/2019/dns-cert/)
```

Comment:
Suggest changing this item to: experimental [DNS certificate management](docs/tasks/security/dns-cert/).